### PR TITLE
loanParams must be BigInt

### LINKED ISSUE #2829: interest error: Cannot mix BigInt and other types, use explicit conversions
## Describe the bug

When the user tries to create a vault by making an offer, there is an error in `interest.js:24`: `Cannot mix BigInt and other types, use explicit conversions`

Error text:

```
TypeError#16: Cannot mix BigInt and other types, use explicit conversions

  at meteredConstructor.makeInterestCalculator (packages/treasury/src/interest.js:24:24)
  at meteredConstructor.makeVaultKit (packages/treasury/src/vault.js:58:45)
  at meteredConstructor.makeLoanKit (packages/treasury/src/vaultManager.js:212:36)

TypeError#16 ERROR_NOTE: Rejection from: (Error#17) : 4866 . 1
TypeError#16 ERROR_NOTE: Sent as error:liveSlots:v16#70001
Nested error under TypeError#16
  Error#17: Event: 4865.1
  
    at Function.applyFunction (packages/tame-metering/src/tame.js:202:19)
    at meteredConstructor.apply (packages/eventual-send/src/E.js:40:44)
    at Alleged: addSeatObj.addSeat (packages/zoe/src/contractFacet/contractFacet.js:456:62)
  
  Error#17 ERROR_NOTE: Caused by: (Error#18)
  Nested error under Error#17
    Error#18: Event: 4864.1
    
      at Function.applyMethod (packages/tame-metering/src/tame.js:184:20)
      at meteredConstructor.deliver (packages/SwingSet/src/kernel/liveSlots.js:525:28)
      at eval (packages/SwingSet/src/kernel/vatManager/deliver.js:51:48)
```

Details of the offer in the wallet:
```
[
    {
        "id": "1617753197369",
        "installationHandleBoardId": "381205908",
        "installationPetname": [
            "Treasury",
            "Installation"
        ],
        "instanceHandleBoardId": "604346717",
        "instancePetname": [
            "Treasury",
            "Instance"
        ],
        "invitation": {},
        "proposalForDisplay": {
            "exit": {
                "onDemand": null
            },
            "give": {
                "Collateral": {
                    "amount": {
                        "brand": {
                            "kind": "brand",
                            "petname": "BLD"
                        },
                        "displayInfo": {
                            "amountMathKind": "nat",
                            "decimalPlaces": 6
                        },
                        "value": 1000000
                    },
                    "purse": {},
                    "pursePetname": "Agoric staking token"
                }
            },
            "want": {
                "RUN": {
                    "amount": {
                        "brand": {
                            "kind": "brand",
                            "petname": "RUN"
                        },
                        "displayInfo": {
                            "amountMathKind": "nat",
                            "decimalPlaces": 6
                        },
                        "value": 18544181
                    },
                    "purse": {},
                    "pursePetname": "Agoric RUN currency"
                }
            }
        },
        "proposalTemplate": {
            "give": {
                "Collateral": {
                    "pursePetname": "Agoric staking token",
                    "value": 1000000
                }
            },
            "want": {
                "RUN": {
                    "pursePetname": "Agoric RUN currency",
                    "value": 18544181
                }
            }
        },
        "requestContext": {
            "channelHandle": {},
            "dappOrigin": "http://localhost:3000",
            "date": 1617753178938,
            "dispatcher": "onOpen",
            "origin": "http://localhost:8000",
            "query": {
                "isQuery": true,
                "suggestedDappPetname": "Treasury"
            },
            "url": "/private/wallet-bridge"
        },
        "status": "accept"
    }
]
```

## To Reproduce
1. Start up the `treasury` app locally. 
2. Create a vault giving 1 display units of BLD and wanting approx 18 display units of RUN (see above in the offer for the smallest denominations)

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
fixes #2829

### CODE REVIEW

File: packages/treasury/src/stablecoinMachine.js [hunk 1]
```diff
@@ -49,6 +50,19 @@ export async function start(zcf) {
     liquidationInstall,
   } = zcf.getTerms();
 
+  assert.typeof(
+    loanParams.chargingPeriod,
+    'bigint',
+    details`chargingPeriod (${q(loanParams.chargingPeriod)}) must be a BigInt`,
+  );
+  assert.typeof(
+    loanParams.chargingPeriod,
```
Comment (michaelfig) [inline packages/treasury/src/stablecoinMachine.js]:
```suggestion
    loanParams.recordingPeriod,
```

Comment (Chris-Hibbert) [reply packages/treasury/src/stablecoinMachine.js]: Thank you Michael.  Fixed.

**Review by michaelfig — APPROVED** [review]
LGTM!

**Review by katelynsills — APPROVED** [review]
Missed this review. Looks good to me!

**Pull request merged**